cluster/saltbase/salt/kubelet/default

```diff
@@ -33,4 +33,12 @@
   {% set docker_root = " --docker_root=" + grains.docker_root -%}
 {% endif -%}
 
-DAEMON_ARGS="{{daemon_args}} {{api_servers}} {{hostname_override}} {{config}} --allow_privileged={{pillar['allow_privileged']}} {{pillar['log_level']}} {{cluster_dns}} {{cluster_domain}} {{docker_root}}"
+{% set network_plugin = "" -%}
+{% set pod_infra = "" -%}
+{% if grains.network_mode == "calico" and 'kubernetes-pool' in grains.roles -%}
+  {% set network_plugin = " --network_plugin=calico" -%}
+  {% set pod_infra = " --pod_infra_container_image=paultiplady/pause" -%}
```

Comment:
I'm not sure about using your "untrusted" pause container. Is there a different one you could use?